apps/src/applab/dropletConfig.js

```diff
@@ -174,6 +187,48 @@ module.exports.blocks = [
   {func: 'setAttribute', parent: api, category: 'Advanced', params: ['"id"', '"scrollHeight"', "200"]},
+
+  {func: 'pinMode', parent: api, category: 'Maker Lab', params: ['13', '"input"']},
+  {func: 'digitalWrite', parent: api, category: 'Maker Lab', params: ['13', '1']},
+  {func: 'digitalRead', parent: api, category: 'Maker Lab', type: 'value', nativeIsAsync: true, params: ['"D4"']},
+  {func: 'analogWrite', parent: api, category: 'Maker Lab', params: ['5', '150']},
+  {func: 'analogRead', parent: api, category: 'Maker Lab', type: 'value', nativeIsAsync: true, params: ['5']},
+
+  {func: 'led', category: 'Circuit', type: 'readonlyproperty', noAutocomplete: true},
+  {func: 'led.on', category: 'Circuit'},
+  {func: 'led.off', category: 'Circuit'},
+  {func: 'led.toggle', category: 'Circuit'},
+  {func: 'led.blink', category: 'Circuit', params: ['50']},
+  {func: 'led.stop', category: 'Circuit'},
+
+  {func: 'on', blockPrefix: 'pixel0.', category: 'Circuit', tipPrefix: '[pixelN]' },
+  {func: 'off', blockPrefix: 'pixel0.', category: 'Circuit', tipPrefix: '[pixelN]' },
+  {func: 'toggle', blockPrefix: 'pixel0.', category: 'Circuit', tipPrefix: '[pixelN]' },
+  {func: 'blink', blockPrefix: 'pixel0.', category: 'Circuit', paletteParams: ['period'], params: ['50'], tipPrefix: '[pixelN]' },
+  {func: 'stop', blockPrefix: 'pixel0.', category: 'Circuit', tipPrefix: '[pixelN]' },
+  {func: 'intensity', blockPrefix: 'pixel0.', category: 'Circuit', params: ['25'], tipPrefix: '[pixelN]' },
+  {func: 'color', blockPrefix: 'pixel0.', category: 'Circuit', paletteParams: ['color'], params: ['"#FF00FF"'], tipPrefix: '[pixelN]' },
+
+  {func: 'pixels', category: 'Circuit', type: 'readonlyproperty', noAutocomplete: true},
+  {func: 'pixels.on', category: 'Circuit'},
+  {func: 'pixels.off', category: 'Circuit'},
+  {func: 'pixels.toggle', category: 'Circuit'},
+  {func: 'pixels.blink', category: 'Circuit', params: ['50']},
+  {func: 'pixels.stop', category: 'Circuit'},
+  {func: 'pixels.intensity', category: 'Circuit', params: ['50']},
+  {func: 'pixels.color', category: 'Circuit'},
+
+  {func: 'piezo', category: 'Circuit', type: 'readonlyproperty', noAutocomplete: true},
+  {func: 'piezo.frequency', category: 'Circuit', params: ['500', '100']},
+  {func: 'piezo.note', category: 'Circuit', params: ['"A4"', '100']},
+  {func: 'piezo.stop', category: 'Circuit'},
+  {func: 'piezo.play', category: 'Circuit', paletteParams: ['song'], params: [JSON.stringify(playSongObject)]},
+
+  {func: 'isPressed', blockPrefix: 'buttonL.', category: 'Circuit', type: 'readonlyproperty', tipPrefix: 'button[L/R]' },
+  {func: 'holdtime', blockPrefix: 'buttonL.', category: 'Circuit', type: 'property', tipPrefix: 'button[L/R]' },
+
+  {func: 'toggleSwitch', category: 'Circuit', type: 'readonlyproperty', noAutocomplete: true},
+  {func: 'toggleSwitch.isOpen', category: 'Circuit', type: 'readonlyproperty' },
```

Comment:
Hmm, on the fence about this. For some recent context, at a high level "Maker Lab" is now being considered more like a set of "Maker APIs" rather than a separate IDE, and will eventually be available from within standard App Lab. Thinking then it might be more akin to how we treat e.g. the Data / Canvas categories and blocks, and maybe not worth separating the definitions out?

Just to think through it, If I were to pull makerlab-specific elements out into separate configs, these blocks / categories might then get required/folded in either in `applab/dropletConfig.js` itself or during `applab.js`'s `Applab.init` initialization before `config.dropletConfig = dropletConfig`. The `commands.js` step definitions might require a bit more refactoring to extract as there are some private helpers in that file.

Or also possible some of this might be able to be managed from within the `BoardController.prototype` at some point during the `applab.js` initialization cycle.

I'll revisit this in the next round of blocks, hoping to have a first set out in time for a Facilitator Summit demo